libsentrykube/quickpatch.py

```diff
@@ -1,15 +1,122 @@
-from typing import Sequence, Mapping
+import os
+from typing import MutableMapping, Optional, Sequence, Union
+import yaml
+import jsonpatch
+from jinja2 import Template
+
+from libsentrykube.service import (
+    get_service_path,
+    get_service_value_overrides_file_path,
+)
+from jsonschema import validate, ValidationError
+
+
+def find_patch_files(service: str, patch: str) -> Optional[str]:
+    """
+    Finds the patch file for the given service and patch name
+    """
+    base_path = get_service_path(service)
+    print(f"Base path: {base_path}")
```

Comment:
please remove the print statements. This is a module that runs the business logic, the presentation is managed by the script in the sentry_kube module.